lib/widget/reusable_stepper.dart

```diff
@@ -12,43 +12,11 @@ class _ReusableStepperState extends State<ReusableStepper> {
         isActive: true,
         state: StepState.indexed,
         content: Text('Clean your hands carefully...')),
-    Step(
```

Comment:
Are these step Widgets no longer needed?

Reply:
Actually the whole `ReusableStepper` is not in use at the moment. But I'm not sure it is there for something else you want to add in the future.

Reply:
Basically this is the call hierarchy
`Routes.sbsGuidance` ->`SBSGuideView` -> `ReusableStepper`
Because `sbsGuidance` in not in use at the moment.